tools/toolchains/golang/build.sh

```diff
@@ -2,6 +2,19 @@
 THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}"  )" >/dev/null 2>&1 && pwd )"
 BUILD_DIR=/tmp/occlum_golang_toolchain
 INSTALL_DIR=/opt/occlum/toolchains/golang
+GO_VERSION="1.18.4"
+
+if [[ $# -ge 1 ]];then
+case $1 in
+          --help|help)
+               echo  "Usage:$0 [go1.16.3|go1.18.4]"
+               exit
+               ;;
+          "go1.16.3"|"1.16.3"|"v1.16.3")
+               GO_VERSION="1.16.3"
+               ;;
```

Comment:
I think we should add the check about 1.18.4.
"go1.18.4"|"1.18.4"|"v1.18.4")
               GO_VERSION="1.18.4"
               ;;

Reply:
ok, I'll add it